src/test/java/org/mule/service/http/impl/service/client/NonBlockingStreamWriterTestCase.java

```diff
@@ -85,11 +85,11 @@ public void isEnabledVariants() throws Exception {
     assertThat(new NonBlockingStreamWriter(1, false).isEnabled(), is(false));
 
     // Default system property is false...
```

Comment:
You will probably want to change this one too.
```suggestion
    // Default system property is true...
```